test/Interpreter/multi_payload_shifting.swift

```diff
@@ -0,0 +1,53 @@
+// RUN: %empty-directory(%t)
+
+// RUN: %target-build-swift -o %t/a.out %s
+// RUN: %target-run %t/a.out 2>&1
```

Comment:
Nitpick: This should use `%target-run-simple-swift`, which is a slightly more flexible substitution. There's also no need to pipe `2>&1`.